espnet/lm/pytorch_backend/lm.py

```diff
@@ -350,21 +350,20 @@ def train(args):
 
     use_sortagrad = args.sortagrad == -1 or args.sortagrad > 0
     # Create the dataset iterators
-    train_iter = ParallelSentenceIterator(train, args.batchsize,
+    batch_size = args.batchsize * max(args.ngpu, 1)
```

Comment:
It is better to show info as the same as ASR.
https://github.com/espnet/espnet/blob/595cce96e1c5f2b71fa58f29deef0827a21d6ea8/espnet/asr/pytorch_backend/asr.py#L365-L366